src/Raven.Server/Documents/Queries/Parser/Lucene/RangeQueryParser.cs

```diff
@@ -3,18 +3,16 @@
 using System.IO;
 using System.Text;
 using System.Text.RegularExpressions;
-
 using Lucene.Net.Analysis;
 using Lucene.Net.Analysis.Tokenattributes;
 using Lucene.Net.Index;
-using Lucene.Net.QueryParsers;
 using Lucene.Net.Search;
 using Raven.Client.Documents.Indexes;
 using Version = Lucene.Net.Util.Version;
 
-namespace Raven.Server.Documents.Queries.Parse
+namespace Raven.Server.Documents.Queries.Parser.Lucene
 {
-    public class RangeQueryParser : QueryParser
+    public class RangeQueryParser : global::Lucene.Net.QueryParsers.QueryParser
```

Comment:
I think we can just use the raw query syntax from Lucene and be done with it.
Loose range queries can be done in RQL